gnupg/_parsers.py

```diff
@@ -1027,6 +1033,9 @@ def sub(self, args):
         subkey = [args[4], args[11]]
         self.curkey['subkeys'].append(subkey)
 
+    def rvk(self, args):
+        self.curuid == None
```

Comment:
I think the revocation packet is on a different UID, at the very end:

{{{
rev:::1:C0A2586F09D77C82:1471422777::::heartsucker <heartsucker@autistici.org>:28x:::::10:
}}}

But I still don't understand why one would reset curuid…